content/en/tracing/setup_overview/open_standards/_index.md

```diff
@@ -385,3 +383,4 @@ Datadog recommends you use the OpenTelemetry Collector Datadog exporter or the O
 [23]: /tracing/setup_overview/open_standards/python#opentelemetry
 [24]: /tracing/setup_overview/open_standards/ruby#opentelemetry
 [25]: /tracing/setup_overview/open_standards/nodejs#opentelemetry
+[26]: https://github.com/DataDog/datadog-agent/blob/7.35.0-rc.1/pkg/config/config_template.yaml#L2911-L3054
```

Comment:
Comment so that we remember to change to `7.35` once released.

Reply:
Worst case scenario, even if it stays like this, I don't think the documentation will be wrong between now and then. Or, I could change it now and practically the link will work after we release.